plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/executor/TesTaskHandler.groovy

```diff
@@ -168,17 +176,31 @@ class TesTaskHandler extends TaskHandler {
     }
 
     protected TesBashBuilder newTesBashBuilder(TaskRun task, String remoteBinDir) {
-        return new TesBashBuilder(task, remoteBinDir)
+        final builder = new TesBashBuilder(task, remoteBinDir)
+        builder.headerScript = "NXF_CHDIR=${Escape.path(WORK_DIR)}"
+        return builder
     }
 
     protected TesTask newTesTask() {
         if( !task.container )
             throw new ProcessUnrecoverableException("Process `${task.lazyName()}` failed because the container image was not specified")
 
         final exec = new TesExecutorModel()
-        exec.command = List.of('/bin/bash', '-c', "${BashWrapperBuilder.BASH} ${wrapperFile.getName()} &> ${TaskRun.CMD_LOG}".toString())
+        String tes_end_point = executor.getEndpoint()
+        if ( tes_end_point.contains('azure.com') ) {
+            String azure_work = WORK_DIR.replace("/", "")
+            final result = new ArrayList(BashWrapperBuilder.BASH)
+            String bashAsString = result.join(' ')
+            exec.command = List.of('/bin/bash', '-c', "${bashAsString} ${azure_work}/${wrapperFile.getName()} &> ${azure_work}/${TaskRun.CMD_LOG}".toString())
+
+        }
+        else 
+            exec.command = List.of('/bin/bash', '-c', "${BashWrapperBuilder.BASH} ${wrapperFile.getName()} &> ${TaskRun.CMD_LOG}".toString())
```

Comment:
@vsmalladi why is this change needed?

Reply:
for our TES it was having issues with the nested list

`/bin/bash -c [/bin/bash -ue] .command.run &> .command.log`

So this change was to bring it back to old way of doing it without changing our Executor too much.

`/bin/bash -c /bin/bash -ue .command.run &> .command.log`

Reply:
I'm talking about specifically this line:
```diff
+            String azure_work = WORK_DIR.replace("/", "")
```

Reply:
Does the container not start in `/work` ?

Reply:
ya on azure we don't start in the work directory

Reply:
where do you start?

Reply:
Our tesRunner, runs at root.

So then i do `NXF_CHDIR=/work` in `.command.run`

Reply:
but the TES task submitted by Nextflow speciffies the work dir as `/work`:
```groovy
        exec.workdir = WORK_DIR
```

so why aren't you respecting that?

Reply:
It has to do how our runner runs in the docker container. I do use it as 

`builder.headerScript = "NXF_CHDIR=${Escape.path(WORK_DIR)}"`

I tried using /work/.command.sh but seems to be an issue what we have access to

Reply:
So you can do `bash work/.command.run` but not `bash /work/.command.run` ?

Reply:
Let me look at this again.

Reply:
> So you can do `bash work/.command.run` but not `bash /work/.command.run` ?

Yep its somehow the runner outside the docker container can access the outfiles. Its super odd. But works.

Reply:
okay I did some cleanup, let's merge this into the main PR and discuss as a whole